apps/yapms/src/lib/components/homepage/SideBarUpdate.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+    export let title: string
+    export let description: string
+</script>
+
+<div class="card card-bordered bg-base-300 shadow-md p-3 mb-3">
+    <div class="text-xl text-center md:text-left font-semibold pb-2">
+        {title}
+    </div>
+    <div class="text-md text-center md:text-left">
+        {description}
+    </div>
+</div>

```

Comment:
*Requests*

1) Could you remove the mb-3 from the card, and wrap the #each loop inside a flex box with gap-3? The parent should be controlling the layout, not the children.


*Suggestions*

1) I think here you could use the daisyUI card-body and card-title classes to control the padding and font, if you want, other wise looks good.

2) I think the md:text-left and text-center can be removed, because the sidebar just disappears before it can shrink below medium.

Reply:
Request #1 and suggestion #2 implemented